Snowflake.Data.Tests/IntegrationTests/ConnectionMultiplePoolsIT.cs

```diff
@@ -17,6 +18,7 @@ namespace Snowflake.Data.Tests.IntegrationTests
     [NonParallelizable]
     public class ConnectionMultiplePoolsIT: SFBaseTest
     {
+        private static readonly SFLogger s_logger = SFLoggerFactory.GetLogger<ConnectionPoolManager>();
```

Comment:
it seems those changes relate to the connection pool feature, can you revert?

Reply:
done